### DISCUSSION TITLE
Parameter types do not match when formatting with %.

### DISCUSSION THREAD

**Discussion by KINGOFKINGOFKEY** [mkdocstrings/mkdocstrings#718] [Documentation]
name = "Alice"
age = 30
message = "My name is %s and I am %d years old." % (name, age)  

plaintext
I'm using the `%` operator for string formatting, but I keep getting a `TypeError` when the types of the arguments don't match the format specifiers. For example:
```python
name = "Alice"
age = "30"  
message = "My name is %s and I am %d years old." % (name, age)

**Comment by pawamoy** [comment]
Sorry, are you sure you posted this in the right repository :smile:? I fail to see how this is relevant to mkdocstrings :slightly_smiling_face: Maybe you just omitted important context?

**Reply by pawamoy** [reply]
Yes seeing your other discussion at https://github.com/mkdocstrings/mkdocstrings/discussions/717, I think you're looking for help on Python programming, not mkdocstrings. Try getting help on [Python Discord](https://discord.com/invite/python), or [Python Discourse forum](https://discuss.python.org). Closing! :relaxed:
